src/components/RabbitMQServerReleaseInfo/index.js

```diff
@@ -89,32 +92,37 @@ export function RabbitMQServerReleaseInfoTable() {
       initialReleaseDate = <></>;
     }
 
-    var endOfSupportDates = [];
-    if (releaseBranch.end_of_community_support) {
-      endOfSupportDates.push(releaseBranch.end_of_community_support);
-    }
-    if (releaseBranch.end_of_commercial_support) {
-      endOfSupportDates.push(releaseBranch.end_of_commercial_support);
+    var endOfSupportDates = ["-", "-"];
+    var hasOSSSupport = false;
+    if (releaseBranch.end_of_support) {
+      /* Release branch is supported. */
+      if (previousReleaseBranch) {
+        const prevReleases = previousReleaseBranch.releases;
+        const initialPrevRelease = prevReleases[prevReleases.length - 1];
+        endOfSupportDates[0] = new Date(initialPrevRelease.release_date);
+      } else {
+        hasOSSSupport = true;
+        endOfSupportDates[0] = "Until next release branch";
```

Comment:
"Until next major or minor release" would be more specific

Reply:
I tried exactly that first, but it's too long for the HTML table we use unfortunately (it breaks the readability of that table).

I plan to rework that table (and not use an HTML table in the first place) once #1710 is merged.

Reply:
I changed it to "Until next minor/major". It's close to your suggestion while still being short to fit the table's constraints. What do you think?